client/src/components/ratings.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StarRatings from 'react-star-ratings';
+
+const Ratings = ({ ratings }) => (
+  <div>
+    <div className="subRatings">
+      <div className="acc-com-cle">
+        <div className="accuacy">Accuracy <StarRatings
```

Comment:
Use css styling in your external stylesheet instead of inline since you have an external stylesheet.

Reply:
I'm not sure what you mean, I didn't do any styling here. Unless you are talking about the StarRatings stuff. Those are props for the StarRatings component and can't go in a style sheet.

Reply:
I didn't realize it was part of the react-star-ratings.  Sorry. Everything else looks good.